### PR TITLE
docs: fix minor issue in local setup section

### PR DESCRIPTION
**Pull request by undef1nd**
## Description

Minor change in Local Setup section of Readme: corrected `make run_local` to `make run` as  there's no `make run_local` command in makefile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,12 +56,12 @@ Durable sync needs only a valid mysql DSN in order to set up connections to a My
 
 To setup a fresh MySQL DB and user: (`mysql -u root`):
 
-    ```
```
Comment (tublitzed) [inline README.md]: Minor nit: unrelated changes?. No big deal for now, we can leave as is, but noting for the future as more changes mean more for folks to review.

Comment (undef1nd) [reply README.md]: Sorry! Got your point.

**Comment by undef1nd** [comment]
@tublitzed I've just made my commit signed

File: README.md (hunk 1 above)
Comment (tublitzed) [reply README.md]: no worries at all

**Comment by undef1nd** [comment]
Merge it, plz, once you have a chance) Thanks!

**Comment by jrconlin** [comment]
Heh, we will, but we have to wait for the CI tests to complete. (No, really. We're bound by that limit too ;) )

And, sadly, begging the CI engine to go faster never works as much as we beg it to.

**Comment by undef1nd** [comment]
Just asking for the future - is it okay that CI takes so long here? Like hours... 😊

**Comment by jrconlin** [comment]
hrm. 13 hours seems like a very long time. Not sure if CI is blocking this or if there's something else preventing CI from running. I may resubmit as a patch from an internal branch.

**Comment by tublitzed** [comment]
@undef1nd I'm going to close this out since we just merged it [here](https://github.com/mozilla-services/syncstorage-rs/pull/474) - In the meantime, we'll work on sorting out our CI woes and hopefully get ahead of your next PR :)

If not, we can take the same workaround until we resolve the issue.

**Comment by pjenvey** [comment]
@undef1nd FYI CI should be working now (w/ its typical slowness of upwards of =~ 45 minutes depending on the time of day) :)

**Comment by undef1nd** [comment]
@pjenvey yay! thanks for letting me know :)

**Pull request closed without merging**